jbmc/regression/jbmc/generics_type_param/test.desc

```diff
@@ -7,6 +7,6 @@ Reading class AWrapper
 Reading class FWrapper
 Reading class IWrapper
 --
-failed to load class \`AWrapper\'
-failed to load class \`FWrapper\'
-failed to load class \`IWrapper\'
+failed to load class \'AWrapper\'
+failed to load class \'FWrapper\'
+failed to load class \'IWrapper\'
```

Comment:
Nit picking: why are we using escapes here?

Reply:
I agree, and it's probably closely enough related to this PR to include it